plugins/Scripts/Image/Hyperstacks/Temporal-Color_Code.ijm

```diff
@@ -140,16 +141,19 @@ macro "Time-Lapse Color Coder" {
 
 function showDialog() {
 	lutA = getList("LUTs");
+	projection = newArray("Max Intensity","Standard Deviation"); //new array for maximum projection methods
  	Dialog.create("Color Code Settings");
 	Dialog.addChoice("LUT", lutA);
 	Dialog.addNumber("start frame", Gstartf);
 	Dialog.addNumber("end frame", Gendf);
+	Dialog.addChoice("Maximum Projection", projection);
```

Comment:
I'd rather call this choice *Projection method* or just *Projection*, as in the case of "Standard Deviation", it *isn't* a maximum projection.

Reply:
All of these are good points. I've made those changes. 
Thanks for the suggestion.